app/browser/api/ledger.js

```diff
@@ -936,10 +947,7 @@ const fetchFavIcon = (publisherKey, url, redirects) => {
     } else if (tail > 0 && (tail + 8 >= blob.length)) return
 
     appActions.onFavIconReceived(publisherKey, blob)
-
-    if (synopsis.publishers && synopsis.publishers[publisherKey]) {
-      synopsis.publishers[publisherKey].faviconURL = blob
-    }
+    savePublisherData(publisherKey, 'faviconURL', blob)
```

Comment:
is this extra save needed? when the action fires, it'll also run the save

Reply:
no need